{
    "uvi": [
        {
            "extracted_data": {
                "advisory_id": "DSA-3544-1",
                "advisory_type": "DSA",
                "affected": [
                    {
                        "distribution_affected": "wheezy",
                        "fixed in": "1.4.5-1+deb7u16"
                    },
                    {
                        "distribution_affected": "jessie",
                        "fixed in": "1.7.7-1+deb8u4"
                    },
                    {
                        "distribution_affected": "stretch",
                        "fixed in": "1.9.4-1"
                    },
                    {
                        "distribution_affected": "sid",
                        "fixed in": "1.9.4-1"
                    }
                ],
                "cve_ids": [
                    "CVE-2016-2513",
                    "CVE-2016-2512"
                ],
                "info_date_string": "2016-04-07",
                "package_name": "python-django",
                "product_name": "Linux",
                "vendor_name": "Debian",
                "vulnerability_status": "yes"
            },
            "meta_data": {
                "uvi_processed_timestamp": "2021-08-03T02:08:37.774720Z",
                "uvi_script_hash": "4d6742cc7fd27472fb19a4c307967e9bd4eebb55e7fcb78430305de2fcc0321848e49403978a5fd94a243249bc017da3919fa53cab26676c5c6b91ab2d5478d4",
                "uvi_script_name": "./www.debian.org-processing.py",
                "uvi_script_version": "0.1.2",
                "uvi_url_processed": "https://www.debian.org/security/2016/dsa-3544"
            }
        }
    ]
}